bin/clean_topics_vs_practices.py

```diff
@@ -0,0 +1,28 @@
+#!python3
+import json
+
+
+def main():
+    with open("config.json") as f:
+        config = json.load(f)
+
+    concepts = [c['slug'] for c in config['concepts']]
```

Comment:
```suggestion
    concepts = {c['slug'] for c in config['concepts']}
```

Making `concepts` a set instead of a list means that searching it is faster: the `in` operator does a linear scan for a list; it can't do a binary search because it can't assume the list is sorted. For a set, it just does a hash comparison.